h/static/scripts/directive/via-dialog.coffee

```diff
@@ -0,0 +1,27 @@
+###*
+# @ngdoc directive
+# @name viaLinkDialog
+# @restrict A
+# @description The dialog that generates a via link to the page h is currently
+# loaded on.
+###
+module.exports = ['$timeout', 'crossframe', ($timeout, crossframe) ->
+    viaUrl = 'https://via.hypothes.is/h/'
```

Comment:
The server side needs to take the URI from somewhere, too.

It's impossible to generate it from the server's main URL, since the VIA deployment is a separate thing.

So we still need an entry in the config file.